examples/requests_adafruit_discord_active_online.py

```diff
@@ -0,0 +1,99 @@
+# SPDX-FileCopyrightText: 2022 DJDevon3 for Adafruit Industries
+# SPDX-License-Identifier: MIT
+# Coded for Circuit Python 8.0
+"""DJDevon3 Adafruit Feather ESP32-S2 Adafruit_Discord_Active_Users_Example"""
+import gc
+import re
+import time
+import ssl
+import json
+import wifi
+import socketpool
+import adafruit_requests
+
+# No user or token required, web scrape.
+
+# Initialize WiFi Pool (There can be only 1 pool & top of script)
+pool = socketpool.SocketPool(wifi.radio)
+
+# Time between API refreshes
+# 900 = 15 mins, 1800 = 30 mins, 3600 = 1 hour
+sleep_time = 900
+
+try:
+    from secrets import secrets
+except ImportError:
+    print("Secrets File Import Error")
+    raise
+
+if sleep_time < 60:
+    sleep_time_conversion = "seconds"
+    sleep_int = sleep_time
+elif 60 <= sleep_time < 3600:
+    sleep_int = sleep_time / 60
+    sleep_time_conversion = "minutes"
+elif 3600 <= sleep_time < 86400:
+    sleep_int = sleep_time / 60 / 60
+    sleep_time_conversion = "hours"
+else:
+    sleep_int = sleep_time / 60 / 60 / 24
+    sleep_time_conversion = "days"
+
+# https://img.shields.io/discord/327254708534116352.svg
+ADA_DISCORD_SVG = "https://img.shields.io/discord/327254708534116352.json"
+
+# Connect to Wi-Fi
+print("\n===============================")
+print("Connecting to WiFi...")
+requests = adafruit_requests.Session(pool, ssl.create_default_context())
+while not wifi.radio.ipv4_address:
+    try:
+        wifi.radio.connect(secrets["ssid"], secrets["password"])
+    except ConnectionError as e:
+        print("Connection Error:", e)
+        print("Retrying in 10 seconds")
+    time.sleep(10)
+    gc.collect()
+print("Connected!\n")
+
+while True:
+    try:
+        print("\nAttempting to GET DISCORD SVG!")  # --------------------------------
+        # Print Request to Serial
+        debug_request = True  # Set true to see full request
+        if debug_request:
+            print("Full API GET URL: ", ADA_DISCORD_SVG)
+        print("===============================")
+        try:
+            ada_SVG_response = requests.get(ADA_DISCORD_SVG).json()
```

Comment:
this one can get rid of svg in the name of the variable as well. it could switch to json, or use 'discord' so that it's not tied to a specific type

Reply:
Ah yes I originally started naming everything SVG and about halfway through found the JSON file. Forgot to rename everything. Will do. BTW I never did figure out how to work with the SVG directly. Typed in .json at the end of the filename  in a fit of hair pulling desperation... and like a miracle it actually worked. Felt like I'd found a hidden cheat code. :P